### PR TITLE
Container login

### LINKED ISSUE #2: markup logging page
Create logging page and  add styles.
login discord

This our model.
discord

### PR DESCRIPTION
**Pull request by paulotijero**
Resolve #2 
even to finish the background #21

### CODE REVIEW

File: public/login.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  </head>
+  <body>
+    <div class="logo">
+      Discord
+    </div>
+    <div class="login">
+      <div class="container-img">
+        <div class="group-img">
+          <img
+            src="https://discordapp.com/assets/0e291f67c9274a1abdddeb3fd919cbaa.png"
```
Comment (diegotc86) [inline public/login.html]: We should download this image later.

Comment (paulotijero) [reply public/login.html]: Yes, I think it would be better

File: public/login.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      </div>
+      <form action="">
+        <span class="username">USERNAME</span>
+        <br />
+        <input
+          class="input-username"
+          type="text"
+          name="username"
+          placeholder="How do you want to be called?"
+          title="You can change this later."
```
Comment (Linzeur) [inline public/login.html]: I don't think about it, I like it, but if we don't add the functionality of changing name later, I think we should take away that.

Comment (paulotijero) [reply public/login.html]: In the original it's, maybe we don't build that functionality but it's only as a visual effect
prueba

**Pull request merged**